src/Registry/Registry.jl

```diff
@@ -163,6 +164,8 @@ function download_registries(io::IO, regs::Vector{RegistrySpec}, depot::String=d
     populate_known_registries_with_urls!(regs)
     regdir = joinpath(depot, "registries")
     isdir(regdir) || mkpath(regdir)
+    FileWatching.mkpidlock(joinpath(regdir, ".pid"), stale_age = 10) do
```

Comment:
Perhaps `.pidlock` or `.lock` as extension? Is there a standard naming?

Reply:
I originally had something more descriptive but changed it to .pid because it appears to be the standard https://stackoverflow.com/questions/8296170/what-is-a-pid-file-and-what-does-it-contain

Reply:
Yea, I thought there were more info in the file here about the lock, but looks like it is jsut the pid.